fetch.bs

```diff
@@ -3763,11 +3921,41 @@ steps:
 
  <li><p>Wait for either <var>response</var>'s <a for=response>body</a> to be null, or
  <var>response</var>'s <a for=response>body</a>'s <a for=body>stream</a> to be
- <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and then set
- <var>request</var>'s <a>done flag</a>.
- <!-- This is really bad and needs to be handled differently at some point. -->
+ <a for=ReadableStream>closed</a> or <a for=ReadableStream>errored</a>, and perform the following steps:
+ <ol>
+  <li><p>Set <var>request</var>'s <a>done flag</a>.
+   <!-- This is really bad and needs to be handled differently at some point. -->
+ </ol>
 </ol>
 
+<p>To <dfn>handle response end</dfn>, given <a for=/>fetch params</a> <var>fetchParams</var> and
```

Comment:
This is only called for network responses. Does that mean that if a service worker handles a response there's no RT entry?